### PR TITLE
Remove the stale bot and migrate to no-response

### PR DESCRIPTION
**Pull request by j-f1**
`no-response` seems to better fit our issue management style — triage issues when they’re opened, and only close issues if the OP hasn’t replied to our request for more information.

Since one of the [guidelines](https://probot.github.io/docs/best-practices/#avoid-the-uncanny-valley) is to avoid the bot pretending to be human, I’d like some 👀 on the comment (specifically the `I’m` part), which is mostly copied from the [`no-response`](https://github.com/desktop/issue-triage/blob/master/canned-messages/no-response.md) canned message.

---

Repository admin steps:

- [x] [Uninstall](https://github.com/apps/stale/installations/new) `stale`
- [x] [Install](https://github.com/apps/no-response/installations/new) `no-response`

`no-response` won’t do anything until the config file is added.

### CODE REVIEW

**Comment by j-f1** [comment]
# stale  → no-response

File: .github/no-response.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+# TODO: also close `needs-reproduction` issues (blocked by https://github.com/probot/no-response/issues/11)
+responseRequiredLabel: more-information-needed
+# Comment to post when closing an issue due to lack of response. 
+closeComment: >
+  Thank you for your issue!
+  
+  We haven’t gotten a response to our questions above.
+  With only the information that is currently in the issue,
+  we don’t have enough information to take action.
+  I’m going to close this but don’t hesitate to reach out
```
Comment (joshaber) [inline .github/no-response.yml]: Let's switch the "I'm" for a "We're" here, so it's phrased more like the team taking action and less a robot taking action 🙅‍♂️ 🤖

**Comment by joshaber** [comment]
`stale` uninstalled and `no-response` installed 👍

File: .github/no-response.yml (hunk 1 above)
Comment (j-f1) [reply .github/no-response.yml]: ✅ Changed

**Comment by joshaber** [comment]
Thanks! 💖

**Comment by joshaber** [comment]
A whole bunch of issues were closed too soon (see https://github.com/probot/no-response/issues/12) so I've uninstalled the bot until we figure out what happened.

**Comment by lee-dohm** [comment]
Taking a look.

**Pull request merged**